### PR TITLE
Sass 932

### PR DESCRIPTION
**Pull request by lz1kwn**
### Description
Created page to include errors and routing

Add a link to the relevant story in Jira
https://jira.tools.tax.service.gov.uk/browse/SASS-932

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of master?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current master that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of master?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

**Comment by matt712** [comment]
uk.gov.hmrc:play-frontend-hmrc could be bumped to 0.86.0-play-28

File: app/controllers/employment/AddEmploymentController.scala [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          case _ => Future.successful(Redirect(appConfig.incomeTaxSubmissionOverviewUrl(taxYear)))
+        }
+      })
+  }
+
+  private def redirectOrRenderView(taxYear:Int)(f: => Future[Result])(implicit user:User[_]): Future[Result] = {
+    inYearAction.notInYear(taxYear) {
+      employmentSessionService.getPriorData(taxYear).flatMap {
+        case Right(IncomeTaxUserData(Some(_))) => Future(Redirect(controllers.employment.routes.EmploymentSummaryController.show(taxYear)))
+        case Right(IncomeTaxUserData(None)) => f
```
Comment (matt712) [inline app/controllers/employment/AddEmploymentController.scala:83]: I'd maybe rename "f" here. It kind of makes sense, but it's hard to read and looks weird

**Review by matt712 — COMMENTED** [review]
Test coverage also. seems like it's slightly down from master. I'm not sure why

File: app/common/SessionValues.scala [hunk 2]
```diff
@@ -21,4 +21,5 @@ object SessionValues {
   val CLIENT_NINO = "ClientNino"
 
   val TAX_YEAR = "TAX_YEAR"
+  val EMPLOYMENT_ID = ""
```
Comment (BenCosfordHMRC) [inline app/common/SessionValues.scala:24]: could this be updated to have the name be the same as the value and be more explicit that it's a temporary employment id in session. 

So could be val TEMP_NEW_EMPLOYMENT_ID = "TEMP_NEW_EMPLOYMENT_ID"

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

**Review by matt712 — APPROVED** [review]

**Comment by BenCosfordHMRC** [comment]
> Test coverage also. seems like it's slightly down from master. I'm not sure why!

[Uploading Screenshot 2021-07-29 at 14.01.57.png…]()

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**